importer/RuRu.py

```diff
@@ -0,0 +1,107 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class RuRu(Monument):
+
+    def set_adm_location(self):
```

Comment:
With [T174934](https://phabricator.wikimedia.org/T174934) it should be possible to get to the Qid directly from `district_id` if that has been added on Wikivoyage. The current structure can be used as a fallback.